### PR TITLE
Create 2018-03-21-mwc-7k-2018-concludes.md

### PR DESCRIPTION
**Pull request by mangomizer**
Blah juan no bully pls

### CODE REVIEW

File: news/2018-03-21-mwc-7k-2018-concludes.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Led by veteran player [jakads](https://osu.ppy.sh/users/259972), Team South Korea showed unparalleled consistency throughout the tournament, with notable performances from [Kim_GodSSI](https://osu.ppy.sh/users/2218047) and [Estonians](https://osu.ppy.sh/users/7014697). They will continue to defend their title for another year, once again proving that they are the dominant force in the osu!mania 7K community.
+	
+Team South Korea will take home the grand prize - $150 per team member, profile badge, and the highly coveted "osu!mania Champion" user title for one year!
+
+**Coming in 2nd place, Team Malaysia walks away proudly with silver!**
+
+Team Captain Cryolien(https://osu.ppy.sh/users/1626983) led his team with unshakable conviction, resulting in their truly splendid performance. A special shoutout to [cheewee10](https://osu.ppy.sh/users/4477497) for his godly performance that could surpass even jakads at times - an impressive feat for his World Cup debut. Well done to ZYuan(https://osu.ppy.sh/users/3337688) and [[MY]Idiot](https://osu.ppy.sh/users/2059742) as well, for holding their own towards the later stages.
```
Comment (MegaApplePi) [inline news/2018-03-21-mwc-7k-2018-concludes.md]: :eyes:

Comment (MegaApplePi) [reply news/2018-03-21-mwc-7k-2018-concludes.md]: - `Cryolien(https://osu.ppy.sh/users/1626983)` -> `[Cryolien](https://osu.ppy.sh/users/1626983)`
- `ZYuan(https://osu.ppy.sh/users/3337688)` -> `[ZYuan](https://osu.ppy.sh/users/3337688)`

**Comment by Ephemeralis** [comment]
dont merge until winner banners and badges have been awarded plz

**Comment by TPGPL** [comment]
btw make more meaningful commits 👀

**Comment by mangomizer** [comment]
Seems weird to delay an announcement just for badges to be distributed. I should also add that the newspost states that the players WILL be awarded profile badges, etc. I would've thought faster = better, but I guess not. Oh well, I've done my part, so w/e :D

**Pull request merged**